examples/mnist/run.py

```diff
@@ -170,22 +168,22 @@ def receive_output(self, logits: tf.Tensor) -> tf.Operation:
       local_data_file="./data/test.tfrecord")
 
   # get model parameters as private tensors from model owner
-  params = tfe.define_private_input(model_owner.player_name,
-                                    model_owner.provide_input, masked=True)  # pylint: disable=E0632
-
-  # we'll use the same parameters for each prediction so we cache them to
-  # avoid re-training each time
+  params = [tfe.define_private_variable(x) for x in model_owner.provide_input()]  # pylint: disable=E0632
+  
   cache_updater, params = tfe.cache(params)
 
+  model = tfe.keras.Sequential()
```

Comment:
we should define the model under `with tfe.protocol.SecureNN():` in order to use the exact relu instead of the approximated one.